go.mod

```
@@ -0,0 +1,12 @@
+module gitlab.com/giantswarm/app
+
+go 1.13
+
+require (
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
```

```suggestion
	sigs.k8s.io/yaml v1.2.0
```

Can we use this instead? https://github.com/giantswarm/fmt/blob/master/go/yaml_marshalling.md

---

Done